lexik/jwt-authentication-bundle/2.3/Makefile

```diff
@@ -0,0 +1,5 @@
+generate-rsa-keys:
+	mkdir -p etc/jwt
+	openssl genrsa -passout pass:${JWT_PASSPHRASE} -out etc/jwt/private.pem -aes256 4096
```

Comment:
To provide a good user experience, we could use here the same strategy as Symfony (https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/3.3/Makefile) to display a good error message when the `openssl` command is not installed in the local computer.